php/class-fieldmanager-group.php

```diff
@@ -148,6 +148,14 @@ public function __construct( $label = '', $options = array() ) {
 				throw new FM_Developer_Exception( esc_html__( 'You cannot use `serialize_data => false` with `index => true`', 'fieldmanager' ) );
 			}
 
+			// A post can only have one parent, so if this saves to post_parent and
+			// it's repeatable, we're doing it wrong.
+			if ( $element->datasource && ! empty( $element->datasource->save_to_post_parent ) && $this->is_repeatable() ) {
```

Comment:
Should this be `$element->is_repeatabe()` to check if the child element you're considering on this loop iteration is itself repeatable? You had checked `$this->is_repeatable()` back in class-fieldmanager-field.php